app/src/main/java/com/github/polypoly/app/network/RemoteDB.kt

```diff
@@ -15,7 +17,7 @@ open class RemoteDB(
     root: String
 ) : IRemoteStorage {
```

Comment:
why is this class `open`?

Reply:
No idea, this is @Jharaxus code, however i'll remove it, if we need it we'll change it again.